backend/app/tests/crud/test_prompt_versions.py

```
@@ -0,0 +1,239 @@
+
+import pytest
+from fastapi import HTTPException
+from sqlmodel import Session
+
+from app.core.util import now
+from app.crud.prompt_versions import (
+    create_prompt_version,
+    delete_prompt_version,
+    get_next_prompt_version,
+)
+from app.crud.prompts import create_prompt
```

Ruff (0.12.2)

`app.crud.prompts.create_prompt` imported but unused

Remove unused import: `app.crud.prompts.create_prompt`

(F401)